src/openrct2/network/NetworkBase.cpp

```diff
@@ -3701,7 +3701,7 @@ GameActions::Result::Ptr network_kick_player(NetworkPlayerId_t playerId, bool is
         return std::make_unique<GameActions::Result>(GameActions::Status::Unknown, STR_NONE);
     }
 
-    if (player && player->Flags & NETWORK_PLAYER_FLAG_ISSERVER)
+    if (player->Flags & NETWORK_PLAYER_FLAG_ISSERVER)
```

Comment:
Note: This one could simply be removed since `player` is already checked for `nullptr` in the `if`-statement before this one, which then exits the function.

Reply:
Let’s remove it then.